cfgov/unprocessed/apps/paying-for-college/js/models/constants-model.js

```diff
@@ -21,7 +21,8 @@ const constantsModel = {
       yearOne: 0,
       yearTwo: 0,
       yearThree: 0
-    }
+    },
+    existingDebtRate: 0.07
```

Comment:
Probably doesn't need to be addressed immediately, but should this be managed in the django admin?

Reply:
Yes, it should. This is a stand-in, and we haven't discussed much what this value should be. We can do a fast-follow of adding it to the django admin.